Documentation/Upgrade/InstallTheNewSource/Index.rst

```diff
@@ -144,26 +144,26 @@ Installing the Source Manually
 Go to `http://typo3.org/download/ <http://typo3.org/download/>`_ and download
 the source package of the new TYPO3 version.
 
-Extract the package on your web server and - in your TYPO3 document root -
+Extract the package on your web server and—in your TYPO3 document root,
 adjust the :file:`typo3_src` symlink.
 
 
 What's the Next Step?
 =====================
 
-In case you did a *minor update*, e.g. from TYPO3 9.5.0 to 9.5.1, database
-updates are usually *not* necessary. All you still have to do is to
+In case you performed a *minor update*, e.g. from TYPO3 9.5.0 to 9.5.1, database
```

Comment:
I would call this an upgrade (in the Debian sense), but it is probably "update" elsewhere.